app/views/games/new.html.erb

```diff
@@ -20,7 +18,12 @@
         <td scope="row">
           <div class="field">
             <div class="checkbox-list">
-              <%= form.collection_check_boxes :player_ids, Player.all, :id, :name, {} %>
+              <% Player.all.each do |player| %>
+                <div>
+                  <%= form.check_box :player_ids, { multiple: true }, player.id, nil %>
+                  <%= form.label :player_ids, player.name %>
+                </div>
+              <% end %>
```

Comment:
Why didn't you use ` <%= form.collection_check_boxes :player_ids, Player.all, :id, :name, {} %> do |b| %>` as you did for all the other checkboxes?